### DISCUSSION TITLE
"Quasars Have Surprisingly Big Dark Matter Halos, New Study Finds"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#159] [Gravitation Models]
Hossenfelder on quasars, if "dark matter" exists, which it may not.
[Quasars Have Surprisingly Big Dark Matter Halos, New Study Finds](https://youtu.be/5cnonJPjRk8?t=184)

**Comment by bgreenman** [comment]
Sabine Hossenfelder is just expressing a hypothesis, because she is somewhat skeptical of the dark matter concept
